guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,43 @@
+[id="proc_enabling-puppet_{context}"]
+= Enabling Puppet Integration with {Project}
+
+A fresh installation of {Project} does not have any Puppet integration configured by default.
+You need to enable the integration as is appropriate for your situation.
+{Project} can be configured to manage and deploy Puppet Server on the {ProjectServer} or on a {SmartProxy}.
```

Comment:
```suggestion
{Project} can be configured to manage and deploy Puppet Server on {ProjectServer} or on {SmartProxy}.
```
I know it seems unnatural but there is a convention for Satellite in particular not to use the article before Satellite and Capsule, so we don't use it.